api/v1alpha1/onload_types.go

```diff
@@ -4,25 +4,118 @@
 package v1alpha1
 
 import (
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// Currently unimplemented
+type KernelBuild struct {
+}
+
+type KernelMapping struct {
+	// Regexp is a regular expression that is used to match against the kernel
+	// versions of the nodes in the cluster
+	Regexp string `json:"regexp"`
+
+	// KernelImage is the image that contains the out-of-tree kernel modules
+	// used by onload, including the sfc driver.
+	KernelImage string `json:"kernelImage"`
```

Comment:
`KernelImage` could be misconstrued as a full kernel not just kernel _module_. I think we should always include the word 'module' or 'driver'. KMM calls this DriverContainer or just ContainerImage, but I'm not partial provided we're consistent.